### DISCUSSION TITLE
Support for `document.queryCommandSupported`

### DISCUSSION THREAD

**Discussion by Lemour-sudo** [capricorn86/happy-dom#1372] [Q&A]
I get the following error when running tests (through vitest on happy-dom) when running my frontend application which depends on the microsoft/vscode library:
`TypeError: document.queryCommandSupported is not a function`

Is there any support for this function as the UI renders without any errors in the Chrome and Firefox browsers?

**Comment by capricorn86** [comment]
@Lemour-sudo this has been fixed now:
https://github.com/capricorn86/happy-dom/issues/1411
